packages/perseus/src/widget-ai-utils/matcher/matcher-ai-utils.test.tsx

```diff
@@ -128,7 +127,13 @@ describe("Matcher AI utils", () => {
                             "Left - Four",
                             "Left - Five",
                         ],
-                        right: shuffledRightItems,
+                        right: [
```

Comment:
I think this was a mistake with the test itself, it seemed to use user input twice whereas I think it was wanting the correct order of things. I'll CC Mike.